lib/jekyll-redirect-from/redirector.rb

```diff
@@ -69,7 +69,11 @@ def redirect_url(site, item)
     end
 
     def redirect_prefix(site)
-      config_github_url(site) || config_baseurl(site) || ""
+      config_url(site) || config_github_url(site) || config_baseurl(site) || ""
```

Comment:
If I specify `url` and `baseurl`, I'd expect them to be joined in this case. `github.url` contains both, so then if it exists, it can be safely used.

https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/